test/controllers/concerns/embedded_app_test.rb

```diff
@@ -75,15 +75,23 @@ def redirect_to_embed
     shop = "my-shop.myshopify.com"
     host = Base64.encode64("#{shop}/admin")
     get :redirect_to_embed, params: { host: host }
-    assert_redirected_to "https://#{shop}/admin/apps/#{ShopifyApp.configuration.api_key}"
+    assert_redirected_to "https://#{shop}/admin/apps/#{ShopifyApp.configuration.api_key}/redirect_to_embed"
   end
 
   test "#redirect_to_embed_app_in_admin redirects to the embed app in the admin when the shop param is present" do
     ShopifyApp.configuration.embedded_app = true
 
     shop = "my-shop.myshopify.com"
     get :redirect_to_embed, params: { shop: shop }
-    assert_redirected_to "https://#{shop}/admin/apps/#{ShopifyApp.configuration.api_key}"
+    assert_redirected_to "https://#{shop}/admin/apps/#{ShopifyApp.configuration.api_key}/redirect_to_embed"
+  end
+
+  test "#redirect_to_embed_app_in_admin keeps original path and params when redirecting to the embed app" do
+    ShopifyApp.configuration.embedded_app = true
+
+    shop = "my-shop.myshopify.com"
+    get :redirect_to_embed, params: { shop: shop, foo: "bar" }
```

Comment:
Could we add these extra params to test that it [doesn't get included](https://github.com/Shopify/shopify_app/pull/1869/files#diff-ac5f4b969898cd49605b9d34ccb2d15486a7ede28a99560265795785ca908a81R29) in the redirect?
```suggestion
    get :redirect_to_embed, params: { shop: shop, foo: "bar", host: "host", id_token:"id_token" }
```

Reply:
Yes, updated